### PR TITLE
[DOC] New Redis Enterprise summary page

### PR DESCRIPTION
**Pull request by nermiller**
Adds an enterprise summary page

### CODE REVIEW

**Comment by nermiller** [comment]
@banker here's the first pass for your review. Could fix the table if needed.

File: docs/stack/about/redis-enterprise.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+
+---
+
+[Redis Enterprise]() is an enterprise-grade Redis, available both [on-premises](https://redis.com/redis-enterprise-software/overview/) and [in the cloud](https://redis.com/try-free/) (on AWS, GCP, or Azure), that simplifies scaling and multi-tenancy, includes many integrations (e.g., K8s), and provides multiples tiers of support.
+
+Read on to learn about Redis Enterprise and the benefits of its extended offerings.
+
+## About Redis the company
+
+[Redis](https://redis.com/), the company, develops OSS Redis and Redis Stack and also provides Redis Enterprise as a commercial product. Redis Enterprise maintains the simplicity and high performance of Redis, while adding many enterprise-grade capabilities:
```
Comment (chayim) [inline docs/stack/about/redis-enterprise.md:16]:
```
[Redis](https://redis.com/), the company, develops [OSS Redis](https://redis.io/download/#redis-downloads) and [Redis Stack](https://redis.io/download/#redis-stack-downloads) and also provides [Redis Enterprise](https://redis.com/redis-enterprise/advantages/) as a commercial product. Redis Enterprise maintains the simplicity and high performance of Redis, while adding many enterprise-grade capabilities: 
```

File: docs/stack/about/redis-enterprise.md [hunk 2]
```diff
@@ -0,0 +1,176 @@
+---
+title: "Redis Enterprise"
+linkTitle: "Enterprise"
+weight: 7
+description: Learn about Redis Enterprise
+aliases:
+
+---
+
+[Redis Enterprise]() is an enterprise-grade Redis, available both [on-premises](https://redis.com/redis-enterprise-software/overview/) and [in the cloud](https://redis.com/try-free/) (on AWS, GCP, or Azure), that simplifies scaling and multi-tenancy, includes many integrations (e.g., K8s), and provides multiples tiers of support.
```
Comment (chayim) [inline docs/stack/about/redis-enterprise.md:10]:
```
[Redis Enterprise]() is an enterprise-grade Redis, available [on-premises](https://redis.com/redis-enterprise-software/overview/) and [in the cloud](https://redis.com/try-free/) (on AWS, GCP, or Azure). It simplifies scaling and multi-tenancy, and includes many integrations (e.g., K8s), and provides multiples tiers of support.
```

File: docs/stack/about/redis-enterprise.md (hunk 1 above)
Comment (banker) [inline docs/stack/about/redis-enterprise.md:16]:
```suggestion
[Redis](https://redis.com/), the company, develops [OSS Redis](https://redis.io/download/#redis-downloads) and [Redis Stack](https://redis.io/download/#redis-stack-downloads) and also provides [Redis Enterprise](https://redis.com/redis-enterprise/advantages/) as a commercial product. Redis Enterprise maintains the simplicity and high performance of Redis, while adding many enterprise-grade capabilities:  
```

File: docs/stack/about/redis-enterprise.md (hunk 2 above)
Comment (banker) [inline docs/stack/about/redis-enterprise.md:10]:
```suggestion
[Redis Enterprise](https://redis.com/redis-enterprise-software/overview/) is an enterprise-grade Redis, available both [on-premises](https://redis.com/redis-enterprise-software/overview/) and [in the cloud](https://redis.com/try-free/) (on AWS, GCP, or Azure), that simplifies scaling and multi-tenancy, includes many integrations (e.g., K8s), and provides multiples tiers of support.
```

**Review by banker — APPROVED** [review]
LGTM!

**Pull request merged**